integrationTests/vm/wasm/utils.go

```diff
@@ -241,6 +241,7 @@ func (context *TestContext) initVMAndBlockchainHook() {
 	argsBuiltIn := builtInFunctions.ArgsCreateBuiltInFunctionContainer{
 		GasSchedule:               gasSchedule,
 		MapDNSAddresses:           DNSAddresses,
+		MapDNSV2Addresses:         DNSAddresses,
```

Comment:
It has been better do have DNSV2Addresses different as DNSAddresses for a better testing

Reply:
Unit testing is done at vm-common.